README.md

```diff
@@ -90,6 +90,38 @@ ros2 launch l3xz control.py
 **Note**: MikroTik [RBMetal5SHPn](https://mikrotik.com/product/RBMetal5SHPn).
 
 #### Cyphal Configuration
+##### How-to-config via `yakut`
+Configure `can0`:
+```bash
+sudo ./setup_slcan.sh --remove-all --basename can --speed-code 5 /dev/serial/by-id/usb-Zubax_Robotics_Zubax_Babel_*-if00
+```
+Configure `yakut`:
+```bash
+python3 -m pip install yakut
+yakut compile https://github.com/OpenCyphal/public_regulated_data_types/archive/refs/heads/master.zip
```

Comment:
This is no longer needed --- Yakut will now compile everything ad-hoc. The updated setup instructions are given in its README

Reply:
there https://github.com/OpenCyphal/yakut#adding-dsdl-namespaces

Reply:
Nice! But I'm unsure whether or not I should modify the instructions here because they are maximum generic. You can actually copy and paste those commands into your shell and they work.  Using `CYPHAL_PATH` that's no longer the case, because every user can have stored that somewhere else. What do you think?

Reply:
You can prepend CYPHAL_PATH with your path to ensure that it always works:

```sh
export CYPHAL_PATH=".your/path:$CYPHAL_PATH"
...
```

Reply:
Unconvinced. That would make stuff work for me, but also expose my internal systems file organisation to the wider Internet. I think I'll use `CYPHAL_PATH` internally, but leave the documentation as is. Environment variables and path are both state, I'd rather have the online doc as generic as possible.

Reply:
>expose my internal systems file organisation to the wider Internet

How? `.your/path` could point anywhere inside the repository; e.g., to the public regulated data types submodule.

Reply:
I do not have public regulated data types as a submodule anywhere :shrug: . 107-Arduino-Cyphal contains the pre-compiled headers and for using yakut I simply compile them once as described in the set-up guide under discussion.

Reply:
*once / per repository from within which I'd like to use yakut.

Reply:
Okay wait, but if you only need the public regulated data types, then you can simply refer the reader to follow the standard Yakut installation instructions, where this is covered in detail. The compilation command is obsolete already and will be removed soon, which will break these instructions if you keep them as-is.

Reply:
I've tried to find middle-ground, @pavel-kirienko can you please take a look at the current state of the README?

Reply:
Special attention should be given to `setup_yakut.sh`.

Reply:
The instructions look fine but:

1. It is still better to just send the reader to follow the official Yakut instructions because they may change (in fact, likely to) in the future. One reason for that is that the entire installation sequence might be replaced with [something like this](https://github.com/OpenCyphal/yakut/issues/60#issuecomment-1162814405):

```bash
bash -c "$(wget -O - https://raw.githubusercontent.com/OpenCyphal/yakut/main/setup.bash)"
```

2. I would rename `setup_yakut.sh` as `setup_yakut_can.sh` because [Cyphal/UDP is upon us](https://github.com/OpenCyphal/specification/pull/127).

3. `setup_yakut.sh` could be automated further to automatically invoke `setup_slcan.sh` when necessary. I use it all the time and it is super convenient. Example:

```bash
# Common UAVCAN register configuration for testing & debugging.
# Source this file into your sh/bash/zsh session before using Yakut and other UAVCAN tools.

if ! [ -e /sys/class/net/slcan0 ]; then
    # https://gist.github.com/pavel-kirienko/32e395683e8b7f49e71413aebf5e1a89
    sudo setup_slcan -r /dev/serial/by-id/usb-Zubax*Babel*
fi

export UAVCAN__CAN__IFACE='socketcan:slcan0'
if [ -e /sys/class/net/slcan1 ]; then
    export UAVCAN__CAN__IFACE="$UAVCAN__CAN__IFACE socketcan:slcan1"
fi
echo "UAVCAN__CAN__IFACE=$UAVCAN__CAN__IFACE"

export UAVCAN__CAN__MTU=8
export UAVCAN__CAN__BITRATE='1000000 1000000'

export UAVCAN__NODE__ID=$(yakut accommodate)
echo "Auto-selected node-ID for this session: $UAVCAN__NODE__ID"
```

Reply:
I've borrowed from your suggestions and linked to yakut's README, see here: https://github.com/107-systems/l3xz/pull/59/commits/71f29711f71ac03004fd8d28a2b6b9e3cfa82698 .

However, I've left the code at one Babel since that's the configuration I'll usually use.

Initially I wanted simpler and more stream-lined instructions for installing and setting-up yakut because imho yakut's actual README has grown a bit too large to easily find what you're looking for. On the other hand, who's going to use this repo except a very limited clientele? So I suppose linking directly to yakut's README is fine since I've outgrown the need for a simple set-up routine documented locally in the repository.

On further reflection I find that its my design philosophy to have everything to build/run/debug a project within a single repository because I don't like gathering the things I need from various repos.